### ISSUE TITLE
Verifier fails with winRM - inspec command works

### ISSUE THREAD

**Issue by hurricanehrndz** [test-kitchen/test-kitchen#1740]
## :speaking_head: Foreword

Thank for taking the time to fill this bug report fully. Without it we may not be able to fix the bug, and the issue may be closed without resolution.

## :ghost: Brief Description

kitchen verify fails with winRM driver -- specifically checking for user

## Version
Test Kitchen version 2.11.2
GEM
  remote: http://rubygems.org/
  specs:
    activesupport (5.2.5)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (>= 0.7, < 2)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    aws-eventstream (1.1.1)
    aws-partitions (1.442.0)
    aws-sdk-apigateway (1.62.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-apigatewayv2 (1.32.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-applicationautoscaling (1.51.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-athena (1.37.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-autoscaling (1.59.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-batch (1.47.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-budgets (1.38.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudformation (1.50.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudfront (1.49.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudhsm (1.30.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudhsmv2 (1.33.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudtrail (1.34.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudwatch (1.51.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudwatchevents (1.45.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cloudwatchlogs (1.40.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-codecommit (1.42.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-codedeploy (1.40.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-codepipeline (1.44.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cognitoidentity (1.31.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-cognitoidentityprovider (1.50.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-configservice (1.60.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-core (3.113.1)
      aws-eventstream (~> 1, >= 1.0.2)
      aws-partitions (~> 1, >= 1.239.0)
      aws-sigv4 (~> 1.1)
      jmespath (~> 1.0)
    aws-sdk-costandusagereportservice (1.31.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-databasemigrationservice (1.52.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-dynamodb (1.60.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ec2 (1.233.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ecr (1.42.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ecs (1.76.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-efs (1.39.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-eks (1.51.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-elasticache (1.55.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-elasticbeanstalk (1.42.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-elasticloadbalancing (1.31.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-elasticloadbalancingv2 (1.61.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-elasticsearchservice (1.51.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-firehose (1.37.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-glue (1.86.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-guardduty (1.45.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-iam (1.52.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-kafka (1.35.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-kinesis (1.32.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-kms (1.43.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-lambda (1.62.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-organizations (1.58.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ram (1.24.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-rds (1.117.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-redshift (1.59.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-route53 (1.48.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-route53domains (1.30.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-route53resolver (1.25.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-s3 (1.93.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sdk-kms (~> 1)
      aws-sigv4 (~> 1.1)
    aws-sdk-secretsmanager (1.46.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-securityhub (1.42.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-servicecatalog (1.59.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ses (1.38.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-shield (1.36.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-sms (1.29.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-sns (1.39.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-sqs (1.38.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-ssm (1.108.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-states (1.39.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-transfer (1.32.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sigv4 (1.2.3)
      aws-eventstream (~> 1, >= 1.0.2)
    azure_graph_rbac (0.17.2)
      ms_rest_azure (~> 0.12.0)
    azure_mgmt_key_vault (0.17.7)
      ms_rest_azure (~> 0.12.0)
    azure_mgmt_resources (0.18.2)
      ms_rest_azure (~> 0.12.0)
    azure_mgmt_security (0.19.0)
      ms_rest_azure (~> 0.12.0)
    azure_mgmt_storage (0.23.0)
      ms_rest_azure (~> 0.12.0)
    bcrypt_pbkdf (1.1.0)
    builder (3.2.4)
    chef-config (16.12.3)
      addressable
      chef-utils (= 16.12.3)
      fuzzyurl
      mixlib-config (>= 2.2.12, < 4.0)
      mixlib-shellout (>= 2.0, < 4.0)
      tomlrb (~> 1.2)
    chef-telemetry (1.0.29)
      chef-config
      concurrent-ruby (~> 1.0)
    chef-utils (16.12.3)
    coderay (1.1.3)
    concurrent-ruby (1.1.8)
    declarative (0.0.20)
    declarative-option (0.1.0)
    diff-lcs (1.4.4)
    docker-api (2.1.0)
      excon (>= 0.47.0)
      multi_json
    domain_name (0.5.20190701)
      unf (>= 0.0.5, < 1.0.0)
    ed25519 (1.2.4)
    erubi (1.10.0)
    excon (0.79.0)
    faraday (1.3.0)
      faraday-net_http (~> 1.0)
      multipart-post (>= 1.2, < 3)
      ruby2_keywords
    faraday-cookie_jar (0.0.7)
      faraday (>= 0.8.0)
      http-cookie (~> 1.0.0)
    faraday-net_http (1.0.1)
    faraday_middleware (1.0.0)
      faraday (~> 1.0)
    ffi (1.15.0)
    fuzzyurl (0.9.0)
    google-api-client (0.52.0)
      addressable (~> 2.5, >= 2.5.1)
      googleauth (~> 0.9)
      httpclient (>= 2.8.1, < 3.0)
      mini_mime (~> 1.0)
      representable (~> 3.0)
      retriable (>= 2.0, < 4.0)
      rexml
      signet (~> 0.12)
    googleauth (0.14.0)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.4, < 3.0)
      memoist (~> 0.16)
      multi_json (~> 1.11)
      os (>= 0.9, < 2.0)
      signet (~> 0.14)
    gssapi (1.3.1)
      ffi (>= 1.0.1)
    gyoku (1.3.1)
      builder (>= 2.1.2)
    hashie (4.1.0)
    http-cookie (1.0.3)
      domain_name (~> 0.5)
    httpclient (2.8.3)
    i18n (1.8.10)
      concurrent-ruby (~> 1.0)
    inifile (3.0.0)
    inspec (4.31.0)
      faraday_middleware (>= 0.12.2, < 1.1)
      inspec-core (= 4.31.0)
      train (~> 3.0)
      train-aws (~> 0.1)
      train-habitat (~> 0.1)
      train-winrm (~> 0.2)
    inspec-core (4.31.0)
      addressable (~> 2.4)
      chef-telemetry (~> 1.0)
      faraday (>= 0.9.0, < 1.4)
      faraday_middleware (~> 1.0)
      hashie (>= 3.4, < 5.0)
      license-acceptance (>= 0.2.13, < 3.0)
      method_source (>= 0.8, < 2.0)
      mixlib-log (~> 3.0)
      multipart-post (~> 2.0)
      parallel (~> 1.9)
      parslet (>= 1.5, < 2.0)
      pry (~> 0.13)
      rspec (>= 3.9, < 3.11)
      rspec-its (~> 1.2)
      rubyzip (>= 1.2.2, < 3.0)
      semverse (~> 3.0)
      sslshake (~> 1.2)
      thor (>= 0.20, < 2.0)
      tomlrb (>= 1.2, < 2.1)
      train-core (~> 3.0)
      tty-prompt (~> 0.17)
      tty-table (~> 0.10)
    jmespath (1.4.0)
    json (2.5.1)
    jwt (2.2.2)
    kitchen-inspec (2.4.1)
      hashie (>= 3.4, <= 5.0)
      inspec (>= 2.2.64, < 5.0)
      test-kitchen (>= 2.7, < 3)
    kitchen-vagrant (1.8.0)
      test-kitchen (>= 1.4, < 3)
    license-acceptance (2.1.13)
      pastel (~> 0.7)
      tomlrb (>= 1.2, < 3.0)
      tty-box (~> 0.6)
      tty-prompt (~> 0.20)
    little-plugger (1.1.4)
    logging (2.3.0)
      little-plugger (~> 1.1)
      multi_json (~> 1.14)
    memoist (0.16.2)
    method_source (1.0.0)
    mini_mime (1.1.0)
    minitest (5.14.4)
    mixlib-config (3.0.9)
      tomlrb
    mixlib-install (3.12.11)
      mixlib-shellout
      mixlib-versioning
      thor
    mixlib-log (3.0.9)
    mixlib-shellout (3.2.5)
      chef-utils
    mixlib-versioning (1.2.12)
    ms_rest (0.7.6)
      concurrent-ruby (~> 1.0)
      faraday (>= 0.9, < 2.0.0)
      timeliness (~> 0.3.10)
    ms_rest_azure (0.12.0)
      concurrent-ruby (~> 1.0)
      faraday (>= 0.9, < 2.0.0)
      faraday-cookie_jar (~> 0.0.6)
      ms_rest (~> 0.7.6)
    multi_json (1.15.0)
    multipart-post (2.1.1)
    net-scp (3.0.0)
      net-ssh (>= 2.6.5, < 7.0.0)
    net-ssh (6.1.0)
    net-ssh-gateway (2.0.0)
      net-ssh (>= 4.0.0)
    nori (2.6.0)
    os (1.1.1)
    parallel (1.20.1)
    parslet (1.8.2)
    pastel (0.8.0)
      tty-color (~> 0.5)
    pry (0.14.0)
      coderay (~> 1.1)
      method_source (~> 1.0)
    public_suffix (4.0.6)
    representable (3.0.4)
      declarative (< 0.1.0)
      declarative-option (< 0.2.0)
      uber (< 0.2.0)
    retriable (3.1.2)
    rexml (3.2.5)
    rspec (3.10.0)
      rspec-core (~> 3.10.0)
      rspec-expectations (~> 3.10.0)
      rspec-mocks (~> 3.10.0)
    rspec-core (3.10.1)
      rspec-support (~> 3.10.0)
    rspec-expectations (3.10.1)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.10.0)
    rspec-its (1.3.0)
      rspec-core (>= 3.0.0)
      rspec-expectations (>= 3.0.0)
    rspec-mocks (3.10.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.10.0)
    rspec-support (3.10.2)
    ruby2_keywords (0.0.4)
    rubyntlm (0.6.3)
    rubyzip (2.3.0)
    semverse (3.0.0)
    signet (0.15.0)
      addressable (~> 2.3)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.5, < 3.0)
      multi_json (~> 1.10)
    sslshake (1.3.1)
    strings (0.2.1)
      strings-ansi (~> 0.2)
      unicode-display_width (>= 1.5, < 3.0)
      unicode_utils (~> 1.4)
    strings-ansi (0.2.0)
    test-kitchen (2.11.2)
      bcrypt_pbkdf (~> 1.0)
      chef-utils (>= 16.4.35)
      ed25519 (~> 1.2)
      license-acceptance (>= 1.0.11, < 3.0)
      mixlib-install (~> 3.6)
      mixlib-shellout (>= 1.2, < 4.0)
      net-scp (>= 1.1, < 4.0)
      net-ssh (>= 2.9, < 7.0)
      net-ssh-gateway (>= 1.2, < 3.0)
      thor (>= 0.19, < 2.0)
      winrm (~> 2.0)
      winrm-elevated (~> 1.0)
      winrm-fs (~> 1.1)
    thor (1.1.0)
    thread_safe (0.3.6)
    timeliness (0.3.10)
    tomlrb (1.3.0)
    train (3.6.0)
      activesupport (>= 5.2.4.3, < 6.0.0)
      azure_graph_rbac (~> 0.16)
      azure_mgmt_key_vault (~> 0.17)
      azure_mgmt_resources (~> 0.15)
      azure_mgmt_security (~> 0.18)
      azure_mgmt_storage (~> 0.18)
      docker-api (>= 1.26, < 3.0)
      google-api-client (>= 0.23.9, <= 0.52.0)
      googleauth (>= 0.6.6, <= 0.14.0)
      inifile (~> 3.0)
      train-core (= 3.6.0)
      train-winrm (~> 0.2)
    train-aws (0.1.67)
      aws-sdk-apigateway (~> 1.0)
      aws-sdk-apigatewayv2 (~> 1.0)
      aws-sdk-applicationautoscaling (>= 1.46, < 1.52)
      aws-sdk-athena (~> 1.0)
      aws-sdk-autoscaling (>= 1.22, < 1.60)
      aws-sdk-batch (>= 1.36, < 1.48)
      aws-sdk-budgets (~> 1.0)
      aws-sdk-cloudformation (~> 1.0)
      aws-sdk-cloudfront (~> 1.0)
      aws-sdk-cloudhsm (~> 1.0)
      aws-sdk-cloudhsmv2 (~> 1.0)
      aws-sdk-cloudtrail (~> 1.8)
      aws-sdk-cloudwatch (~> 1.13)
      aws-sdk-cloudwatchevents (>= 1.36, < 1.46)
      aws-sdk-cloudwatchlogs (~> 1.13)
      aws-sdk-codecommit (~> 1.0)
      aws-sdk-codedeploy (~> 1.0)
      aws-sdk-codepipeline (~> 1.0)
      aws-sdk-cognitoidentity (>= 1.26, < 1.32)
      aws-sdk-cognitoidentityprovider (>= 1.46, < 1.51)
      aws-sdk-configservice (~> 1.21)
      aws-sdk-core (~> 3.0)
      aws-sdk-costandusagereportservice (~> 1.6)
      aws-sdk-databasemigrationservice (>= 1.42, < 1.53)
      aws-sdk-dynamodb (~> 1.31)
      aws-sdk-ec2 (~> 1.70)
      aws-sdk-ecr (~> 1.18)
      aws-sdk-ecs (~> 1.30)
      aws-sdk-efs (~> 1.0)
      aws-sdk-eks (~> 1.9)
      aws-sdk-elasticache (~> 1.0)
      aws-sdk-elasticbeanstalk (~> 1.0)
      aws-sdk-elasticloadbalancing (~> 1.8)
      aws-sdk-elasticloadbalancingv2 (~> 1.0)
      aws-sdk-elasticsearchservice (~> 1.0)
      aws-sdk-firehose (~> 1.0)
      aws-sdk-glue (>= 1.71, < 1.87)
      aws-sdk-guardduty (~> 1.31)
      aws-sdk-iam (~> 1.13)
      aws-sdk-kafka (~> 1.0)
      aws-sdk-kinesis (~> 1.0)
      aws-sdk-kms (~> 1.13)
      aws-sdk-lambda (~> 1.0)
      aws-sdk-organizations (>= 1.17, < 1.59)
      aws-sdk-ram (>= 1.21, < 1.25)
      aws-sdk-rds (~> 1.43)
      aws-sdk-redshift (~> 1.0)
      aws-sdk-route53 (~> 1.0)
      aws-sdk-route53domains (~> 1.0)
      aws-sdk-route53resolver (~> 1.0)
      aws-sdk-s3 (~> 1.30)
      aws-sdk-secretsmanager (>= 1.42, < 1.47)
      aws-sdk-securityhub (~> 1.0)
      aws-sdk-servicecatalog (>= 1.48, < 1.60)
      aws-sdk-ses (~> 1.0)
      aws-sdk-shield (~> 1.30)
      aws-sdk-sms (~> 1.0)
      aws-sdk-sns (~> 1.9)
      aws-sdk-sqs (~> 1.10)
      aws-sdk-ssm (~> 1.0)
      aws-sdk-states (>= 1.35, < 1.40)
      aws-sdk-transfer (>= 1.26, < 1.33)
    train-core (3.6.0)
      addressable (~> 2.5)
      ffi (!= 1.13.0)
      json (>= 1.8, < 3.0)
      mixlib-shellout (>= 2.0, < 4.0)
      net-scp (>= 1.2, < 4.0)
      net-ssh (>= 2.9, < 7.0)
    train-habitat (0.2.22)
    train-winrm (0.2.12)
      winrm (>= 2.3.6, < 3.0)
      winrm-elevated (~> 1.2.2)
      winrm-fs (~> 1.0)
    tty-box (0.7.0)
      pastel (~> 0.8)
      strings (~> 0.2.0)
      tty-cursor (~> 0.7)
    tty-color (0.6.0)
    tty-cursor (0.7.1)
    tty-prompt (0.23.0)
      pastel (~> 0.8)
      tty-reader (~> 0.8)
    tty-reader (0.9.0)
      tty-cursor (~> 0.7)
      tty-screen (~> 0.8)
      wisper (~> 2.0)
    tty-screen (0.8.1)
    tty-table (0.12.0)
      pastel (~> 0.8)
      strings (~> 0.2.0)
      tty-screen (~> 0.8)
    tzinfo (1.2.9)
      thread_safe (~> 0.1)
    uber (0.1.0)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.7)
    unicode-display_width (2.0.0)
    unicode_utils (1.4.0)
    winrm (2.3.6)
      builder (>= 2.1.2)
      erubi (~> 1.8)
      gssapi (~> 1.2)
      gyoku (~> 1.0)
      httpclient (~> 2.2, >= 2.2.0.2)
      logging (>= 1.6.1, < 3.0)
      nori (~> 2.0)
      rubyntlm (~> 0.6.0, >= 0.6.3)
    winrm-elevated (1.2.3)
      erubi (~> 1.8)
      winrm (~> 2.0)
      winrm-fs (~> 1.0)
    winrm-fs (1.3.5)
      erubi (~> 1.8)
      logging (>= 1.6.1, < 3.0)
      rubyzip (~> 2.0)
      winrm (~> 2.0)
    wisper (2.0.1)

PLATFORMS
  ruby

DEPENDENCIES
  inspec
  kitchen-inspec
  kitchen-vagrant
  test-kitchen

BUNDLED WITH
   2.1.4

## Environment

MacOS 10.15.7, vagrant, rvm, ruby 2.7.2

## Scenario

Running inspec works
```rb
describe port(3389) do
  it { should be_listening }
end

describe user('mysql') do
 it { should_not exist }
end
```
```sh
 inspec exec test/integration/default/default_spec.rb --user vagrant --host 127.0.0.1 --backend winrm  --password vagrant --port 2227

Profile: tests from test/integration/default/default_spec.rb (tests from test.integration.default.default_spec.rb)
Version: (not specified)
Target:  winrm://vagrant@http://127.0.0.1:2227/wsman:3389

  Port 3389
     ✔  is expected to be listening
  User mysql
     ✔  is expected not to exist
```
Running kitchen verify fails
```sh
Profile: tests from {:path=>"/Users/chernand/prj/vagrant_resources/packer-windows/test/integration/default"} (tests from {:path=>".Users.chernand.prj.vagrant_resources.packer-windows.test.integration.default"})
Version: (not specified)
Target:  winrm://vagrant@http://127.0.0.1:2227/wsman:3389

  Port 3389
     ✔  is expected to be listening
  User mysql
     ×  is expected not to exist
     Command `Function ConvertTo-SID { Param([byte[]]$BinarySID)
       (New-Object System.Security.Principal.SecurityIdentifier($BinarySID,0)).Value
     }

     Function Convert-UserFlag { Param  ($UserFlag)
       $List = @()
       Switch ($UserFlag) {
         ($UserFlag -BOR 0x0001) { $List += 'SCRIPT' }
         ($UserFlag -BOR 0x0002) { $List += 'ACCOUNTDISABLE' }
         ($UserFlag -BOR 0x0008) { $List += 'HOMEDIR_REQUIRED' }
         ($UserFlag -BOR 0x0010) { $List += 'LOCKOUT' }
         ($UserFlag -BOR 0x0020) { $List += 'PASSWD_NOTREQD' }
         ($UserFlag -BOR 0x0040) { $List += 'PASSWD_CANT_CHANGE' }
         ($UserFlag -BOR 0x0080) { $List += 'ENCRYPTED_TEXT_PWD_ALLOWED' }
         ($UserFlag -BOR 0x0100) { $List += 'TEMP_DUPLICATE_ACCOUNT' }
         ($UserFlag -BOR 0x0200) { $List += 'NORMAL_ACCOUNT' }
         ($UserFlag -BOR 0x0800) { $List += 'INTERDOMAIN_TRUST_ACCOUNT' }
         ($UserFlag -BOR 0x1000) { $List += 'WORKSTATION_TRUST_ACCOUNT' }
         ($UserFlag -BOR 0x2000) { $List += 'SERVER_TRUST_ACCOUNT' }
         ($UserFlag -BOR 0x10000) { $List += 'DONT_EXPIRE_PASSWORD' }
         ($UserFlag -BOR 0x20000) { $List += 'MNS_LOGON_ACCOUNT' }
         ($UserFlag -BOR 0x40000) { $List += 'SMARTCARD_REQUIRED' }
         ($UserFlag -BOR 0x80000) { $List += 'TRUSTED_FOR_DELEGATION' }
         ($UserFlag -BOR 0x100000) { $List += 'NOT_DELEGATED' }
         ($UserFlag -BOR 0x200000) { $List += 'USE_DES_KEY_ONLY' }
         ($UserFlag -BOR 0x400000) { $List += 'DONT_REQ_PREAUTH' }
         ($UserFlag -BOR 0x800000) { $List += 'PASSWORD_EXPIRED' }
         ($UserFlag -BOR 0x1000000) { $List += 'TRUSTED_TO_AUTH_FOR_DELEGATION' }
         ($UserFlag -BOR 0x04000000) { $List += 'PARTIAL_SECRETS_ACCOUNT' }
       }
       $List
     }

     $Computername = $Env:Computername
     $adsi = [ADSI]"WinNT://$Computername"
     $adsi.Children | where {$_.SchemaClassName -eq 'user'} | ForEach {
       New-Object PSObject -property @{
         uid = ConvertTo-SID -BinarySID $_.ObjectSID[0]
         username = $_.Name[0]
         description = $_.Description[0]
         disabled = $_.AccountDisabled[0]
         userflags = Convert-UserFlag  -UserFlag $_.UserFlags[0]
         passwordage = [math]::Round($_.PasswordAge[0]/86400)
         minpasswordlength = $_.MinPasswordLength[0]
         mindays = [math]::Round($_.MinPasswordAge[0]/86400)
         maxdays = [math]::Round($_.MaxPasswordAge[0]/86400)
         warndays = $null
         badpasswordattempts = $_.BadPasswordAttempts[0]
         maxbadpasswords = $_.MaxBadPasswordsAllowed[0]
         gid = $null
         group = $null
         groups = @($_.Groups() | Foreach-Object { $_.GetType().InvokeMember('Name', 'GetProperty', $null, $_, $null) })
         home = $_.HomeDirectory[0]
         shell = $null
         domain = $Computername
         lastlogin = if($_.lastlogin.getType().Tostring() -eq "System.Management.Automation.PSMethod" ){ $null }else{[String]$_.lastlogin}
       }
     } | ConvertTo-Json
     ` timed out after 0 seconds

Test Summary: 1 successful, 1 failure, 0 skipped
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Verify failed on instance <default-windows-10>.  Please see .kitchen/logs/default-windows-10.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration




```
## Steps to Reproduce

[If you are filing an issue what are the things we need to do in order to repro your problem?]

example:

1. Get Any windows 10 box
2. Write an inspec file to verify existence of user
3. Watch it fail

## Expected Result

Manual verify test should be the same as kitchen verify

## Actual Result

kitchen verify does not work with windows and winrm

**Comment by clintoncwolfe** [comment]
```
timed out after 0 seconds
```
This is the key line.  The user verification relies on an InSpec `command` resource under the hood. In fact, all `command` resources would fail with the same error under kitchen-inspec.

This was a regression in inspec 4.31.0 (https://github.com/inspec/inspec/issues/5454) - we released a fix for it in 4.31.1 . Try updating to that version and try again. 

Thanks for the excellent report!

**Comment by hurricanehrndz** [comment]
Thankyou! Can confirm update works.
